pages.it/common/git-clone.md

```diff
@@ -0,0 +1,24 @@
+# git clone
+
+> Clona un repository esistente.
+> Maggiori informazioni: <https://git-scm.com/docs/git-clone>.
+
+- Clona un repository remoto esistente:
+
+`git clone {{url_repository_remoto}}`
+
+- Clona un repository remoto insieme ai suoi sottomoduli:
+
+`git clone --recursive {{url_repository_remoto}}`
+
+- Clona un repository locale:
+
+`git clone -l`
```

Comment:
```suggestion
`git clone {{percorso/a/repository/locale}}`
```
This is also wrong in the English version. Will make a PR to address it ASAP. The `-l` flag is unnecessary, and the path to the local repository to clone is mandatory.

Reply:
Made EN PR (see #3470).